marimbabot_vision/scripts/generate_data.py

```diff
@@ -44,8 +44,11 @@ def generate_sample(i):
     """Generate a sample and save it to disk"""
     string, staff = generate_piece()
     os.makedirs(f"{OUTPUT_DIR}/{i}", exist_ok=True)
+    header_block = Block(name="header")
+    header_block.tagline = "#ff"
```

Comment:
Is it possible to set the tag line in the same way you set the name?

Reply:
I don't know, this is how they proposed it in the docs.

Reply:
Yes. Before and after below.
![big-staff_1](https://user-images.githubusercontent.com/20795522/212710535-0f1ebaa3-f3d8-4c13-86c4-4d486dad3761.png)
![big-staff_1](https://user-images.githubusercontent.com/20795522/212710544-75c9d6a4-62b1-40d1-aeee-ca12fb64145b.png)